project/frontend/src/components/Modal.js

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 /**
  * A modal window with a fixed position on the viewport.
  *
- * @param {bool} props.open True if the modal should be shown; false if it
+ * @param {boolean} props.open True if the modal should be shown; false if it
```

Comment:
nit but should this be `boolean=` like in likes 20 and 22?

Reply:
The `=` indicates that the param has a default value. I could give the `open` prop a default and add a `=` but semantically I think it makes more sense if open doesn't have a default, so that it's clear the parent component should be telling the modal whether it's open or not explicitly. The `center` and `centerHorizontal` props are by default set to false if they are not set explicitly by the parent component.